ambari-server/src/main/java/org/apache/ambari/server/actionmanager/ActionDBAccessorImpl.java

```diff
@@ -218,8 +218,10 @@ public Request getRequest(long requestId) {
 
     // only request commands which actually need to be aborted; requesting all
     // commands here can cause OOM problems during large requests like upgrades
+    List<HostRoleStatus> statusesToAbort = new ArrayList<>(HostRoleStatus.SCHEDULED_STATES);
```

Comment:
its better to create new constant for SCHEDULED_STATES + SCHEDULED_STATES instead of combining them each time

Reply:
Added and tested the requested changes.